client/components/lists/listBody.js

```diff
@@ -35,11 +35,40 @@ BlazeComponent.extendComponent({
       sortIndex = Utils.calculateIndex(lastCardDom, null).base;
     }
 
+    // Parse for @user and #label mentions, stripping them from the title
+    // and applying the appropriate users and labels to the card instead.
+    const currentBoard = Boards.findOne(Session.get('currentBoard'));
+
+    // Find all @-mentioned usernames, collect a list of their IDs and strip
+    // their mention out of the title.
+    let foundUserIds = []; // eslint-disable-line prefer-const
```

Comment:
By the way I agree that this ESlint rule isn’t right, I will probably remove it. Actually the following is perfectly valid in ES6:

```es6
const list = [];
list.push('abc');
```

The only thing that is prevented by the `const` usage is re-assigning the value of `list`, which would raise an error. But actual mutation is still possible... that’s confusing to say the least.

Don’t modify this line, I just wanted to share my thoughts on it ;-)